config/routes.rb

```diff
@@ -1,3 +1,9 @@
 Rails.application.routes.draw do
-  devise_for :user
+  mount_devise_token_auth_for 'User', at: 'auth'
+  
+  get 'home/index'
+  get 'user/index'
+  
+  root to: "home#index"
```

Comment:
With my previous comment I meant we don't need any of these routes
```
get 'home/index'
get 'user/index'
root to: "home#index"
```
Lets just keep the mount of devise token auth!

Reply:
My bad ... What happened is that github only painted me two lines